simple-chat/package.json

```
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "scripts": {
+    "start": "webpack-dev-server --mode=development --hot",
+    "build": "webpack --mode=development"
```

Сборка проекта должна быть с модом `production`.

```suggestion
    "build": "webpack --mode=production"
```